vm/proxyapp/proxyappclient.go

```diff
@@ -96,7 +99,9 @@ func (p *pool) init(params *proxyAppParams, cfg *Config) error {
 	if cfg.Command != "" {
 		p.proxy, err = runProxyApp(params, cfg.Command, usePipedRPC)
 	} else {
-		p.proxy = &ProxyApp{}
+		p.proxy = &ProxyApp{
+			useTCP: useTCPRPC,
```

Comment:
Let's use some new "transferFilesContent" config param from "cfg." instead of useTCPRPC?
It will enable all the pipes/tcp + paths/content combinations.
Additionally your specific configuration will have more chances to survive in case of any refactoring.

And to be sure your configuration will not be affected by some future refactoring (if any), tests that cover your configuration [happy path](https://en.wikipedia.org/wiki/Happy_path) may be helpful.
To add tests or not is up to you.

Reply:
I made the change to add and use a transfer_file_content field to the Config.

I haven't found a good way to test the behavior yet, so I'll leave that for later.